resources/dicts/events/injury/mountainous.json

```diff
@@ -117,5 +117,40 @@
                 "reg_death": "m_c died from {PRONOUN/m_c/poss} wounds after falling from a tall cliff."
         }
         ]
-    }    
+    },
+    {
+        "event_id": "mtn_injury_war_cliffsaved",
+        "location": ["mountainous"],
+        "season": ["any"],
+        "sub_type": ["war"],
+        "weight": 20,
+        "event_text": "m_c acts without thinking, yanking the o_c_n warrior {PRONOUN/m_c/subject}'d been fighting back from the edge of the cliff and saving their life.",
+        "m_c": {
+            "status": ["warrior", "deputy", "leader", "apprentice"],
+            "age": ["adolescent", "young adult", "adult", "senior adult"]
+        },
+        "injury": [
+            {
+                "cats": [
+                    "m_c"
+                ],
+                "injuries": [
+                    "battle_injury"
```

Comment:
not sure this is the best injury to use here, as the event text doesn't actually imply direct injury to the cat.  i think it might actually be nice to give a lower stakes injury, as opposed to lots of the more serious ones we're giving, maybe the cat just wrenches a muscle in the struggle to pull the enemy cat back onto solid ground?  giving the "sore" condition?